model/feature-flag/registry.yaml

```
@@ -9,26 +9,126 @@ groups:
       - id: feature_flag.key
         type: string
         stability: experimental
-        brief: The unique identifier of the feature flag.
+        brief: The lookup key of the feature flag.
         examples: ["logo-color"]
-      - id: feature_flag.provider_name
+      - id: feature_flag.system
         type: string
         stability: experimental
-        brief: The name of the service provider that performs the flag evaluation.
+        brief: Identifies the feature flag provider.
         examples: ["Flag Manager"]
       - id: feature_flag.variant
         type: string
         stability: experimental
         examples: ["red", "true", "on"]
         brief: >
-          SHOULD be a semantic identifier for a value. If one is unavailable, a
-          stringified version of the value can be used.
+          A semantic identifier for an evaluated flag value.
         note: |-
           A semantic identifier, commonly referred to as a variant, provides a means
           for referring to a value without including the value itself. This can
           provide additional context for understanding the meaning behind a value.
           For example, the variant `red` maybe be used for the value `#c05543`.
-
-          A stringified version of the value can be used in situations where a
-          semantic identifier is unavailable. String representation of the value
-          should be determined by the implementer.
+      - id: feature_flag.context.id
+        type: string
+        stability: experimental
+        examples: ["5157782b-2203-4c80-a857-dbbd5e7761db"]
+        brief: >
+          The unique identifier for the flag evaluation context. For example, the targeting key.
+      - id: feature_flag.version
+        type: string
+        stability: experimental
+        examples: ["1", "01ABCDEF"]
+        brief: >
+          The version of the ruleset used during the evaluation. This may be any stable value which uniquely identifies the ruleset.
+      - id: feature_flag.set.id
+        type: string
+        stability: experimental
+        examples: ["proj-1", "ab98sgs", "service1/dev"]
+        brief: >
+          The identifier of the [flag set](https://openfeature.dev/specification/glossary/#flag-set) which the feature flag belongs to in a flag management system.
+      - id: feature_flag.reason
```

@beeme1mr do you think `feature_flag.reason` is clear that it is talking specifically about the evaluation or should we do evaluation reason? In other semconv it looks like they are a bit more verbose about this.

```suggestion
      - id: feature_flag.evaluation.reason
```

---

I think evaluation reason is certainly more clear. :thinking:

---

We called it `feature_flag.evaluation.reason` in the Google doc. @dyladan error code and message also contained the `feature_flag.evaluation` prefix.